indico/web/client/js/legacy/libs/timetable/Draw.js

```diff
@@ -202,9 +202,12 @@ type(
         if (this.eventData.location) {
           this.locationDiv.append(addComma ? ', ' : '');
           this.locationDiv.append(this.eventData.location);
-        }
-
-        // If it's a contribution add the speakers information
+        } 
+      	this.vcdiv = Html.div('timetableBlockVC');
+	if (this.eventData.vcrooms) {
```

Comment:
Something went wrong with the indentation here. Accidental use of tabs instead of spaces?

Reply:
Should be fixed now